sdk/samples/iot/paho_iot_hub_c2d_sample.c

```diff
@@ -244,23 +245,24 @@ static void disconnect_mqtt_client_from_iot_hub(void)
 
 static void parse_c2d_message(
     char* topic,
-    int topic_len,
-    const MQTTClient_message* message,
-    az_iot_hub_client_c2d_request* c2d_request)
+    int const topic_len,
```

Comment:
Same here/elsewhere. Will stop mentioning.

```suggestion
    int topic_len,
```

Reply:
yeah, that was the definition signature for the declaration above